lib/tools/common/patching_parallel.py

```
@@ -0,0 +1,1244 @@
+    This allows independent patches in the same worker to succeed even if
+    an earlier patch failed (they might not actually depend on each other).
+
+    Args:
+        worker_items: List of PatchWorkItem objects to process sequentially
+
+    Returns:
+        List of ParallelPatchResult objects
+    """
+    worker_results = []
+    previous_group_id = None
+    for i, work_item in enumerate(worker_items):
```

Ruff (0.15.20)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)